docs/source/geo_def.rst

```diff
@@ -1,41 +1,41 @@
 Geometry definitions
 ====================
-The module **pyresample.geometry** contains classes for describing different kinds of types
-of remote sensing data geometries. The use of the different classes is described below.
+The module :mod:`pyresample.geometry <geometry>` contains classes for describing different kinds
+of types of remote sensing data geometries. The use of the different classes is described below.
 
 Remarks
 -------
 
-All longitudes and latitudes provided to **pyresample.geometry** must be in degrees.
-Longitudes must additionally be in the [-180;+180[ validity range.
+All longitudes and latitudes provided to :mod:`pyresample.geometry <geometry>` must be
+in degrees. Longitudes must additionally be in the [-180;+180[ validity range.
 
-As of version 1.1.1, the **pyresample.geometry** contructors will check the range of
-longitude values, send a warning if some of them fall outside validity range,
+As of version 1.1.1, the :mod:`pyresample.geometry <geometry>` contructors will
+check the range of longitude values, send a warning if some of them fall outside validity range,
 and automatically correct the invalid values into [-180;+180[.
 
-Use function **utils.wrap_longitudes** for wrapping longitudes yourself.
+Use function :mod:`utils.wrap_longitudes <utils.wrap_longitudes>` for wrapping longitudes yourself.
 
 AreaDefinition
 --------------
 
-The cartographic definition of grid areas used by Pyresample is contained in an
-object of type AreaDefintion. The following arguments are needed to initialize
-an area:
+The cartographic definition of grid areas used by Pyresample is
+contained in an object of type :mod:`AreaDefinition <geometry.AreaDefinition>`
```

Comment:
Does this `mod` work? `AreaDefinition` is not a module.

Reply:
:mod:`AreaDefinition <geometry.AreaDefinition>` does work. However it's probably misleading for people reading the code. I will use "func" for functions and "mod" for modules!